src/App.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import Page from './Page';
+
+export default function App() {
+  const [todoList, setTodoList] = useState([]);
+
+  const onInsert = (text) => {
+    setTodoList((oldArray) => [
+      ...oldArray,
+      { key: todoList.length + 1, text },
+    ]);
+  };
+
+  const onFinish = (key) => {
+    const temp = [...todoList];
+    const idx = temp.findIndex((item) => item.key === key);
+
+    if (idx <= -1) {
+      return;
+    }
+
+    temp.splice(idx, 1);
+    setTodoList(temp);
+  };
+
+  return <Page list={todoList} onInsert={onInsert} onFinish={onFinish} />;
```

Comment:
```suggestion
  return (
    <Page 
      list={todoList}
      onInsert={onInsert}
      onFinish={onFinish} 
    />
  );
```

저는 속성이 2개 이상일 때는 줄바꿈해주면 코드가독성이 더 좋다고 생각하고 코드를 정리해주는 편이에요.
코드 가독성을 고려한 자기만의 코드 정리 기준을 세우시면 좋아요.

Reply:
말씀하신편이 더 깔끔한것 같습니다 :) 감사합니다!

Reply:
린트가 도와주지 않네요 ㅠㅠ

Reply:
속성을 더 추가하니 흩트려주네요,,